hack/windows/Install-OVS.ps1

```diff
@@ -0,0 +1,163 @@
+Param(
+    [parameter(Mandatory = $false)] [string] $DownloadDir,
+    [parameter(Mandatory = $false)] [string] $DownloadURL,
+    [parameter(Mandatory = $false)] [string] $OVSInstallDir = "C:\openvswitch"
+)
+
+$ErrorActionPreference = "Stop"
+$OVSDownloadURL = "https://github.com/vmware-tanzu/antrea/releases/download/v0.7.0/ovs-2.12.90-win64.zip"
```

Comment:
I will have the actual link very soon (it will be something like `downloads.antrea.io/ovs/ovs-2.12.90-win64.zip` hopefully), but we are still working on it. We can merge this PR first and update the link in master later.

Reply:
Cool, thanks @antoninbas .